tox.ini

```diff
@@ -37,21 +36,12 @@ description =
     run tests
     alldeps: with all optional dependencies
     devdeps: with the latest developer version of key dependencies
-    oldestdeps: with the oldest supported version of key dependencies
     cov: and test coverage
-    numpy117: with numpy 1.17.*
-    numpy118: with numpy 1.18.*
-    astropylts: with the latest astropy LTS
 
 # The following provides some specific pinnings for key packages
 deps =
 
-    numpy117: numpy==1.17.*
-    numpy118: numpy==1.18.*
-
-    # We don't support 4.0.x but let's keep this for when LTS changes
-    astropylts: astropy==4.0.*
-    astropylts: gwcs>=0.14,<0.15
+    cov: astropy==5.0.0
```

Comment:
If timeout goes away with this pin, that means whatever causing the timeout went into between `astropy` 5.0.0 and 5.0.1.

Reply:
Nope... not it. 🤯